Upload to MDSplus from windows support added.

seanalsop opened the pull request.

Commit:
Added ability to upload chans to mdsplus.
	modified:   mds_upload.py

	Can now upload channels to MDSplus.

Commit:
Added support for channelised upload on windows.
	modified:   mds_upload.py
	Added a method to upload pre-saved channelised data to
	an MDSplus server. Data channelised by host_demux.py.

Commit:
Added a comment to show how to set env vars on windows.
	modified:   mds_upload.py

Commit:
Do not upload format file for dirfiles.
	modified:   mds_upload.py

petermilne reviewed (changes requested):
Are the segments really 1k (seems really SMALL), in any case, they don't seem to have a size

In the help, please avoid Which What Why :-)

"node to pull data from"  is sufficient
"Whether to upload data as a segment or not"  => "1: upload data as segment".
"Whether to store channelized data to MDSplus." => "1: store channelize data"

Commit:
Made requested changes.
	modified:   mds_upload.py
	Changed incorrect comment,
	Changed incorrect help text.
